.github/workflows/rust.yml

```
@@ -16,12 +16,5 @@ jobs:
     uses: noir-lang/.github/.github/workflows/rust-format.yml@main
 
   spellcheck:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: streetsidesoftware/cspell-action@v2
-        with:
-          files: |
-            **/*.{md,rs}
-          incremental_files_only: true # Run this action on files which have changed in PR
-          strict: false # Do not fail, if a spelling mistake is found (This can be annoying for contributors)
+    name: spellcheck
+    uses: noir-lang/.github/.github/workflows/spellcheck.yml@phated/reusable-spellcheck
```

```suggestion
    uses: noir-lang/.github/.github/workflows/spellcheck.yml@main
```
Once https://github.com/noir-lang/.github/pull/5 is merged